src/rules/memberAccessRule.ts

```diff
@@ -96,6 +96,13 @@ function walk(ctx: Lint.WalkContext<Options>) {
                 }
             }
         }
+        if (isConstructorDeclaration(node)) {
```

Comment:
you can move this check into the `for...of` loop above and use `child` instead of `node`. A `ConstructorDeclaration` can only occur in `ClassLikeDeclaration.members`.

You can also add `&& child.body !== undefined` to avoid checking constructor overload signatures.